src/shared/views/BuildingFootprint/FootprintDisplay.js

```diff
@@ -78,14 +115,44 @@ export default class FootprintDisplay extends React.Component<Props> {
         gestureState: GestureState,
     ): boolean => Math.abs(gestureState.dx) > this.swipeThreshold;
 
+    bounceImage = (direction: string) => {
+        // bounce the image left or right when the user tries to swipe past what
+        // they're allowed to, to give them some visual feedback
+        const { animatedMarginLeft, animatedMarginRight } = this.state;
+        let value;
+        if (direction === 'left') {
+            value = animatedMarginRight;
+        } else {
+            value = animatedMarginLeft;
+        }
+        Animated.sequence([
+            Animated.timing(value, {
+                toValue: 100,
+                duration: 100,
+                useNativeDriver: false,
+            }),
+            Animated.timing(value, {
+                toValue: 0,
+                duration: 100,
+                useNativeDriver: false,
+            }),
+        ]).start();
+    };
+
     handlePanResponderEnd = (event: PressEvent, gestureState: GestureState) => {
         // swipe completed, decide what to do
         const { nextTask, previousTask } = this.props;
         const swipeMinLength = 0.2;
         if (gestureState.dx < -GLOBAL.TILE_VIEW_HEIGHT * swipeMinLength) {
```

Comment:
coudl you add a comment here? I think it defines the threshold for swiping?

Reply:
Added!